pubspec.yaml

```diff
@@ -7,7 +7,7 @@ homepage: https://github.com/dart-lang/dartdoc
 environment:
   sdk: '>=1.14.0 <2.0.0'
 dependencies:
-  analyzer: 0.27.3-alpha.7
+  analyzer: 0.27.4-alpha.8
```

Comment:
Great.  Thanks!

I've got this fixed in the SDK (as of https://github.com/dart-lang/sdk/commit/fba5cbc425e54b015d9ffefac4e0789d5502846c) and the next dev build should be good.  Sorry again for the oversight!